c_src/cb.c

```diff
@@ -651,6 +655,8 @@ ERL_NIF_TERM cb_http(ErlNifEnv* env, handle_t* handle, void* obj)
     ErlNifBinary value_binary;
     enif_alloc_binary(cb.ret.size, &value_binary);
     memcpy(value_binary.data, cb.ret.data, cb.ret.size);
+    free(cb.ret.data);
+    free(cb.ret.key);
```

Comment:
It appears to be allocated in lcb_make_http_request along with `data`.  In my case, a major leak was caused by the `data` part.  The other four pieces of data didn't contribute that much.